targets/simple_switch/sample_extern/macsec/README.md

```diff
@@ -0,0 +1,25 @@
+# ADD CRYPTO EXTERN to SIMPLE_SWITCH
+
+For reference, the extern is implemented in crypto.cpp in this directory.
+This cpp code is to be incorporated in
+behavioral-model/targets/simple_switch/simple_switch.cpp
```

Comment:
That's not accurate. You should instead include the following instructions:
* configure bmv2 with the `--enable-modules` flag so that extern definitions can be provided at runtime
* build the macsec extern as a shared library by running `make` inside this directory
* when starting `simple_switch`, load the shared library with `--load-modules libexterncrypto.so`

Please also make sure that these instructions work properly.

Reply:
I don't see a `--enable-modules` as an arg to simple_switch.  How do I configure it then?